proxy/src/coco/source.rs

```diff
@@ -42,8 +42,27 @@ pub struct Person {
     pub avatar: String,
 }
 
-/// Representation of a code commit.
+/// Commit statistics.
+#[derive(Serialize)]
+pub struct CommitStats {
+    /// Additions.
+    pub additions: u64,
+    /// Deletions.
+    pub deletions: u64,
+}
+
+/// Representation of a changeset between two revs.
 pub struct Commit {
+    /// The commit header.
+    pub header: CommitHeader,
+    /// The change statistics for this commit.
+    pub stats: CommitStats,
+    /// The changeset introduced by this commit.
+    pub diff: surf::diff::Diff,
```

Comment:
So this is always the `Diff` between this `Commit` and its parent?

Reply:
Yeah, exactly, since it's on a `Commit` object.